docs/docs/reference/index.md

```diff
@@ -4,9 +4,9 @@ title: Overview
 
 This section of the documentation contains the API reference for all the official packages under the `@auth/*` and `@next-auth/*` scopes.
 
-## Roadmap
+## Integrations
```

Comment:
Need to update the link here:

https://github.com/nextauthjs/next-auth/blob/05bf014603bd8cc1f648b46771482d94ed7fd8c8/docs/docs/getting-started/introduction.mdx#L24